uiowa/uiowa
# [IWP] Writer bio

Resolves https://github.com/uiowa/uiowa/issues/7549





# How to test

```
ddev blt ds --site=iwp.uiowa.edu && ddev drush @iwp.local config:import --source ../config/sites/iwp.uiowa.edu --partial -y && ddev drush @iwp.local cim -y && ddev drush @iwp.local en devel_generate && ddev drush @iwp.local devel-generate:content 40 --bundles writer_bio && ddev drush @iwp.local uli /
```
1. Add a session year `1996`
2. Add a writer bio entry: https://iwp.uiowa.ddev.site/node/add/writer_bio
3. Confirm countries and languages are multiselect
4. Confirm writing sample files render with 'Writing Sample Original Language' and `Writing Sample`
5. Confirm taxonomy is rabbit holed for anonymous users
6. Review content type creation checkboxes below in details


<details>

- [x] Node type
- [x] Fields (can any existing base fields be used? field_image (featured image), body, field_teaser)
- [x] Use body field with summary enabled
- [x] Default display
        - We have been defaulting to using layout builder to set these up. At a bare minimum, it is recommended that there is a separate header (title, breadcrumbs) and content sections to handle spacing correctly. Currently there is a hack to establish the layout in config but then copy/paste/cim an existing layout display that contains a header section for consistency.
- [x] Teaser display
- [x] Form display
- [x] Disable promote to front page unless your custom content type needs it.
- [x] Sticky if you plan on using it.
- [x] Revisions should be max of 10
- [x] Preview settings (don't require)
- [x] Sitemap enabled
- [x] Path alias pattern
- [x] Allowed in main menu?
- [x] Metatags (add the existing field field_meta_tags to the content type, configure a separate metatag bucket)
- [ ] Schema.org Blueprints (map new and existing fields to a schema.org type and properties)
- [x] Should it be allowed to be referenced in the Featured Content block? (if so, field.field.block_content.featured_content.field_featured_content_item will need a partial split)
- [x] Moderation workflow - copy the settings from our standard content types
        - For moderation control to be added to a Layout Builder content template without causing errors, the form display for the content type needs to be re-saved (for some reason). After that, the moderation control block should be able to be placed without issue.
        - After syncing the site, cim and cr in order to get the moderation workflow block to work.
- [x] Role permissions
- [x] Consider using _sitenow_node_form_defaults() - This puts certain shared fields in the sidebar.

</details>
